### ISSUE TITLE
NordVPN still blocked despite executable and service in exceptions

### ISSUE THREAD

**Issue by albertosottile** [pylorak/TinyWall#29]
I tried to manually whitelist all the involved process and even connect via NordVPN + autolearn every once in a while, but it does not solve the issue. After 2-3 days, TinyWall starts again to block the outgoing connections.

Settings:
![settings](https://github.com/pylorak/TinyWall/assets/7440277/0c943c5b-1d90-4115-befd-343a20751cb0)

Exception:
![exception](https://github.com/pylorak/TinyWall/assets/7440277/2d0c90ea-33b5-4657-92dd-82eeaed34970)

Connections
![blocked](https://github.com/pylorak/TinyWall/assets/7440277/0e3168ba-d6ac-4f50-ad4e-07c4ad20726f)
 blocked:

**Comment by kvvsi** [comment]
Seeing similar issues with [Cloudflare WARP](https://cloudflarewarp.com/).

**Comment by lacertrader** [comment]
Similar issue with qBittorrent: "C:\Program Files\qBittorrent\qbittorrent.exe" whitelisted with "No Restrictions" defined but a lot of connections blocked.

**Comment by xroberx** [comment]
Same things happens with ExpressVPN, there's another issue opened (#14) about this exact same problem.

**Comment by 9ao9ai9ar** [comment]
Is it still being blocked if you check both the "VPN L2TP/IPSec" and "VPN PPTP" checkboxes under "Optional"?

![{6CB01E7E-C02B-40B5-ADDE-2FD1F79002EB}](https://github.com/user-attachments/assets/1644658c-142c-4af6-90f3-5258b28ba10f)

**Comment by 9ao9ai9ar** [comment]
Might be related to [this issue](https://github.com/pylorak/TinyWall/issues/6), where the proposed "fixes" range from enabling "Unblock LAN traffic" to reinstalling TinyWall.

![Screenshot 2024-07-30 230246](https://github.com/user-attachments/assets/ea5a5b4e-ddc2-46b4-9f8c-c61410193ed9)

**Comment by 9ao9ai9ar** [comment]
See if #47 solves your issue. In short, make a "No restrictions" firewall exception to "System" and reboot Windows.

**Comment by realgeekyjeff** [comment]
I had same issue with ProtonVPN but creating an exception for both it's service and executable helped me fix the issue. Now it's connecting.
